src/shared.ts

```diff
@@ -101,12 +101,16 @@ export const codeIncludesLibraryName = (code: string, libList: ImpConfig['libLis
 export const addImportToCode = (code: string, impConfig: ImpConfig, removeoldImport = false) => {
 
   const { importMaps, codeRemoveOriginImport } = parseImportModule(code, impConfig.libList)
-  
+
   let importStr = ''
 
   impConfig.libList.forEach(({libName, style}) => {
     if (importMaps[libName]) {
       importMaps[libName].forEach(item => {
+        if (libName == "element-plus"){
+          item = item.replace(item[0],item[0].toLowerCase());
```

Comment:
I think line 111 & line112 can replace with  below
Because [`paramCase`](https://www.npmjs.com/package/param-case) method has already been import in line 5 at this file
This will make the code more readable than using regexp twice.

```suggestion
         item = paramCase(item)
```